diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x '*.c' -x '*.h' -x doc -x include origsrc/pnet-0.7.4/configure.in src/pnet-0.7.4/configure.in
--- origsrc/pnet-0.7.4/configure.in	2006-01-14 23:17:01.000000000 -0600
+++ src/pnet-0.7.4/configure.in	2006-08-10 12:42:34.948000000 -0500
@@ -340,7 +340,7 @@
 dnl Collect up the libraries that we need for termcap access.
 AC_SUBST(TERMCAPLIBS)
 TERMCAPLIBS=""
-if ! test "x$THREADS" = "xwin32" ; then
+if ! test "x$THREADS" = "xwin32" -a "x$cygwin" = "xfalse" ; then
 	TERMINFOLIBS=""
 	maybe_termcap=no
 	maybe_terminfo=no
