--- origsrc/pnet-0.7.4/configure.in	2006-01-14 23:17:01.000000000 -0600
+++ src/pnet-0.7.4/configure.in	2006-08-10 12:42:34.948000000 -0500
@@ -340,7 +340,7 @@
 dnl Collect up the libraries that we need for termcap access.
 AC_SUBST(TERMCAPLIBS)
 TERMCAPLIBS=""
-if ! test "x$THREADS" = "xwin32" ; then
+if ! test "x$THREADS" = "xwin32" -a "x$cygwin" = "xfalse" ; then
 	TERMINFOLIBS=""
 	maybe_termcap=no
 	maybe_terminfo=no
